gammapy/data/observation_summary.py

```diff
@@ -0,0 +1,132 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import absolute_import, division, print_function, unicode_literals
+import numpy as np
+from astropy.units import Quantity
+from astropy.coordinates import SkyCoord, Angle
+
+from .observation import ObservationTable
+
+__all__ = [
+    'ObservationTableSummary',
+]
+
+class ObservationTableSummary(object):
+    """Observation table summary.
+
+    Class allowing to summarize informations conatained in 
+    Observation index table (`~gammapy.data.ObservationTable`)
+
+    Parameters:
+    -----------
+    obs_table:  `~gammapy.data.ObservationTable`
+        Observation index table
+    target_pos: `~astropy.coordinates.SkyCoord`
+        Target position
+    """
+
+    def __init__(self, obs_table, target_pos=None):
+        self.obs_table = obs_table
+        self.target_pos = target_pos
+
+    @property
+    def offset(self):
+        """
```

Comment:
For this I would suggest a shorter docstring as described here:
http://docs.gammapy.org/en/latest/development/howto.html#functions-or-class-methods-that-return-a-single-object

```python
@property
def offset(self):
    """Offset of observations relative to the target position (`astropy.coordinates.Angle`)."""
```